src/main/java/org/folio/rs/service/ConfigurationsService.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.rs.service;
+
+import org.folio.rs.domain.dto.StorageConfiguration;
+import org.folio.rs.domain.dto.StorageConfigurations;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.CachePut;
+import org.springframework.cache.annotation.Cacheable;
+
+public interface ConfigurationsService {
```

Comment:
Please don't do separate interface and Impl. You don't need interface is there is supposed to be only one class of this interface. Spring don't need it.